dsls/sql/src/main/java/org/apache/beam/dsls/sql/utils/CalciteUtils.java

```diff
@@ -76,27 +76,27 @@ public static Integer toJavaType(SqlTypeName typeName) {
   /**
    * Get the {@code SqlTypeName} for the specified column of a table.
    */
-  public static SqlTypeName getFieldType(BeamSqlRecordType schema, int index) {
+  public static SqlTypeName getFieldType(BeamSqlRowType schema, int index) {
     return toCalciteType(schema.getFieldsType().get(index));
   }
 
   /**
-   * Generate {@code BeamSqlRecordType} from {@code RelDataType} which is used to create table.
+   * Generate {@code BeamSqlRowType} from {@code RelDataType} which is used to create table.
    */
-  public static BeamSqlRecordType toBeamRecordType(RelDataType tableInfo) {
+  public static BeamSqlRowType toBeamRowType(RelDataType tableInfo) {
     List<String> fieldNames = new ArrayList<>();
     List<Integer> fieldTypes = new ArrayList<>();
     for (RelDataTypeField f : tableInfo.getFieldList()) {
       fieldNames.add(f.getName());
       fieldTypes.add(toJavaType(f.getType().getSqlTypeName()));
     }
-    return BeamSqlRecordType.create(fieldNames, fieldTypes);
+    return BeamSqlRowType.create(fieldNames, fieldTypes);
   }
 
   /**
    * Create an instance of {@code RelDataType} so it can be used to create a table.
    */
-  public static RelProtoDataType toCalciteRecordType(final BeamSqlRecordType that) {
+  public static RelProtoDataType toCalciteRecordType(final BeamSqlRowType that) {
```

Comment:
public static RelProtoDataType toCalcite**Row**Type(final BeamSqlRowType that) {